cla_public/apps/scope/views.py

```diff
@@ -71,6 +72,13 @@ def add_link(choice):
 
         return render_template("scope/diagnosis.html", choices=display_choices, nodes=nodes)
 
+    def get_category_for_larp(self, session):
+        category_for_LARP = session.checker.category
```

Comment:
I would move this into an `else` statement on the `if`, so that we're only assigning this value if there's no category match, rather than setting it and then maybe overloading it

Reply:
👍